src/main/java/de/dennisguse/opentracks/io/file/importer/AbstractFileTrackImporter.java

```diff
@@ -104,11 +104,6 @@
         this.recordingDistanceInterval = PreferencesUtils.getRecordingDistanceInterval(context);
     }
 
-    AbstractFileTrackImporter(Context context, ContentProviderUtils contentProviderUtils, Track.Id importTrackId) {
-        this(context, contentProviderUtils);
-        this.importTrackId = importTrackId;
```

Comment:
Attribute must be removed completely (it is always null) and we have an `if (importTrackId == null)` on this class.